1/a-burke/project1.py

```diff
@@ -0,0 +1,105 @@
+#Author: Ally Burke
+
+import sys
+sys.path.append('../../extra-files')
+sys.path.append('../../0/a-burke')
+from molecule import Molecule
+import numpy as np
+from numpy import linalg as la
+from masses import mass
+from masses import charge
+import numpy.lib.scimath as nls
+import scipy.constants as sci
+import math
+
+#Read in hessian matrix
+h = open('../../extra-files/hessian.dat').read()
```

Comment:
Sometimes you will encounter errors if you open files this way. It's often safer to use something like:

with open('') as r:
    r.read()

because this will close that file as soon as you're done with it.